cmd/cape/cmd/encrypt.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/capeprivacy/cli/sdk"
+)
+
+var encryptCmd = &cobra.Command{
+	Use:   "encrypt",
+	Short: "Encrypt data.",
+	Long: "Encrypt input data, takes data as the first and only argument.\n" +
```

Comment:
You can use raw string literals for multi line strings to make this a bit more readable ...

```golang
Long: `Encrypt input data, takes data as the first and only argument.
Encrypt can also read input data .... etc
`,
```